build/build.go

```diff
@@ -87,7 +101,28 @@ func Import(path string, mode build.ImportMode, installSuffix string, buildTags
 		}
 	}
 
-	return pkg, nil
+	jsFiles, err := jsFilesFromDir(pkg.Dir)
+	if err != nil {
+		return nil,err
+	}
+
+	return &PackageData{Package: pkg, JSFiles: jsFiles}, nil
+}
+
+// ImportDir is like Import but processes the Go package found in the named
+// directory.
+func ImportDir(path string, mode build.ImportMode) (*PackageData, error) {
```

Comment:
`path` should be renamed to `dir`, just like with https://golang.org/pkg/go/build/#Context.ImportDir